TestBangazonAPI/BangazonAPITests.cs

```diff
@@ -0,0 +1,101 @@
+using Newtonsoft.Json;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using BangazonAPI;
+using BangazonAPI.Models;
+namespace BangazonAPITests
+{
+    public class CustomerTests
+    {
+        [Fact]
+        public async Task GetCustomer_Success()
+        {
+
+            using (var client = new APIClientProvider().Client)
+            {
+                var response = await client.GetAsync("api/Customer");
+                response.EnsureSuccessStatusCode();
```

Comment:
Please verify that you got at least one `Customer` back from the API.

Reply:
fixed